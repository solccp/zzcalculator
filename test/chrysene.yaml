---
structure_id: 1
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 16]
- [18, 17]
- [15, 2]
- [3, 10]
- [4, 7]
- [9, 8]
- [11, 12]
- [13, 14]
---
structure_id: 2
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 16]
- [18, 17]
- [15, 2]
- [3, 4]
- [10, 9]
- [7, 11]
- [8, 14]
- [12, 13]
---
structure_id: 3
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 16]
- [18, 17]
- [15, 2]
- [3, 4]
- [10, 9]
- [7, 8]
- [11, 12]
- [13, 14]
---
structure_id: 4
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 16]
- [18, 17]
- [15, 2]
- [3, 4]
- [10, 9]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 5
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 16]
- [18, 17]
- [15, 2]
- [11, 12]
- [13, 14]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 6
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 2]
- [15, 17]
- [16, 18]
- [3, 10]
- [4, 7]
- [9, 8]
- [11, 12]
- [13, 14]
---
structure_id: 7
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 2]
- [15, 17]
- [16, 18]
- [3, 4]
- [10, 9]
- [7, 11]
- [8, 14]
- [12, 13]
---
structure_id: 8
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 2]
- [15, 17]
- [16, 18]
- [3, 4]
- [10, 9]
- [7, 8]
- [11, 12]
- [13, 14]
---
structure_id: 9
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 2]
- [15, 17]
- [16, 18]
- [3, 4]
- [10, 9]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 10
number_of_atoms: 18
double_bonds:
- [6, 5]
- [1, 2]
- [15, 17]
- [16, 18]
- [11, 12]
- [13, 14]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 11
number_of_atoms: 18
double_bonds:
- [6, 5]
- [3, 10]
- [4, 7]
- [9, 8]
- [11, 12]
- [13, 14]
rings:
- [1, 2, 15, 17, 18, 16]
---
structure_id: 12
number_of_atoms: 18
double_bonds:
- [6, 5]
- [3, 4]
- [10, 9]
- [7, 11]
- [8, 14]
- [12, 13]
rings:
- [1, 2, 15, 17, 18, 16]
---
structure_id: 13
number_of_atoms: 18
double_bonds:
- [6, 5]
- [3, 4]
- [10, 9]
- [7, 8]
- [11, 12]
- [13, 14]
rings:
- [1, 2, 15, 17, 18, 16]
---
structure_id: 14
number_of_atoms: 18
double_bonds:
- [6, 5]
- [3, 4]
- [10, 9]
rings:
- [1, 2, 15, 17, 18, 16]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 15
number_of_atoms: 18
double_bonds:
- [6, 5]
- [11, 12]
- [13, 14]
rings:
- [1, 2, 15, 17, 18, 16]
- [3, 4, 7, 8, 9, 10]
---
structure_id: 16
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [16, 18]
- [17, 15]
- [2, 3]
- [10, 9]
- [7, 11]
- [8, 14]
- [12, 13]
---
structure_id: 17
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [16, 18]
- [17, 15]
- [2, 3]
- [10, 9]
- [7, 8]
- [11, 12]
- [13, 14]
---
structure_id: 18
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [16, 18]
- [17, 15]
- [2, 3]
- [10, 9]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 19
number_of_atoms: 18
double_bonds:
- [10, 9]
- [15, 17]
- [16, 18]
- [7, 11]
- [8, 14]
- [12, 13]
rings:
- [1, 6, 5, 4, 3, 2]
---
structure_id: 20
number_of_atoms: 18
double_bonds:
- [10, 9]
- [15, 17]
- [16, 18]
- [7, 8]
- [11, 12]
- [13, 14]
rings:
- [1, 6, 5, 4, 3, 2]
---
structure_id: 21
number_of_atoms: 18
double_bonds:
- [10, 9]
- [15, 17]
- [16, 18]
rings:
- [1, 6, 5, 4, 3, 2]
- [7, 8, 14, 13, 12, 11]
