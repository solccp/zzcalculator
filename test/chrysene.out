 8 + 10 x + 3 x^2
 total: 21
